### ISSUE TITLE
Possible Default Solver Issue

### ISSUE THREAD

**Issue by ccoffrin** [dwavesystems/dwave-cloud-client#18]
Can you reproduce this on v0.2.2?

`.dwrc` along these lines,
```
name|url,token
```

python3 interpreter,
```
conn = dwave_micro_client.Connection()
solver = conn.get_solver()
...
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/site-packages/dwave_micro_client.py", line 320, in get_solver
    self.solvers[data['id']] = Solver(self, data)
  File "/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/site-packages/dwave_micro_client.py", line 646, in __init__
    self.id = data['id']
TypeError: list indices must be integers or slices, not str
```

**Comment by randomir** [comment]
Thanks for reporting this. We're working on replacing the legacy `.dwrc` all together. The new version will handle profiles and authentication in a more user-friendly way.

**Comment by randomir** [comment]
But, FWIW, I can't reproduce this with `dwave-micro-client==0.2.2` and Python 3.5.2 on Linux.

**Comment by randomir** [comment]
@ccoffrin Looks like the last line of that traceback is missing. Is it maybe `TypeError: list indices must be integers or slices, not str`? That could mean the response you received from API was ill-formed. Pull request #24 adds some checks for that case.

**Comment by ccoffrin** [comment]
You are correct, I fixed the note above.  I was able to reproduce on os x, python 3.5.  I believe the root of the issue is that no default solver is specified.

**Comment by randomir** [comment]
Fixed as of #103 as part of #101.
